R/paths.R

```diff
@@ -64,5 +64,6 @@ find_file <- function(file_path) {
     }
   }
 
-  stop("Couldn't find the file ", file_path, " in any of the mock directories.")
+  error_msg <- glue::glue("Couldn't find the file {file_path} in any of the mock directories.")
+  rlang::abort(error_msg)
```

Comment:
```suggestion
  rlang::abort(glue("Couldn't find the file {file_path} in any of the mock directories."))
```

I _think_ this is still under the 100 character lintr line limit, but would be good to confirm